README.md

```diff
@@ -41,19 +41,25 @@ There is also an option to forward the requests to use Google's [Data Loss Preve
 
 Example:
 ```js
-var redactor = require('redact-pii')({enableGoogleCloudDLP : true, googleCloudDLPOptions: {credentials : {client_email: 'client_email', private_key: 'api_key'}}});
+var redactor = require('redact-pii')({enableGoogleCloudDLP : true, googleCloudDLPOptions: {clientOptions: {credentials : {client_email: 'client_email', private_key: 'api_key'}}}});
 redactor.redact('我的卡号是1234-5678-9876-5432').then(res => {
     console.log(res);
     //我的卡号是CREDIT_CARD_NUMBER
 });
 ```
-This redacts of an exhaustive list of languages (Chinese, German,etc) and exhaustive list of [PII fields](https://cloud.google.com/dlp/docs/infotypes-reference)
+This redacts an exhaustive list of languages (Chinese, German,etc) and exhaustive list of [PII fields](https://cloud.google.com/dlp/docs/infotypes-reference)
 
 If Google's DLP redaction throws an error (i.e. "RESOURCE_EXHAUSTED: Quota exceeded for quota metric") then by default it will fallback to use the standard redaction. You can turn off this default fallback logic by setting the `disableDLPFallbackRedaction` option to true.
 ```js
 var redactor = require('redact-pii')({enableGoogleCloudDLP : true, disableDLPFallbackRedaction: true});
 ```
 
+Advanced options to customize Google DLP behavior are also available in `googleCloudDLPOptions` :
```

Comment:
Maybe also note that these are inside `inspectOverrides`.. so `googleCloudDLPOptions.inspectOverrides`?

Reply:
Yeah good idea.  I also just realized I need to update the docs for the `disableDLPFallbackRedaction` option since I moved it to `googleCloudDLPOptions.disableFallbackRedaction`